src/MyHealth/DayIndicator.tsx

```diff
@@ -0,0 +1,145 @@
+import React, { FunctionComponent } from "react"
+import { View, Text, ViewStyle, TextStyle, StyleSheet } from "react-native"
+import dayjs from "dayjs"
+
+import { DateTimeUtils } from "../utils"
+import { Outlines, Colors, Typography, Spacing } from "../styles"
+import { CheckInStatus, DayLogData } from "./symptoms"
+
+interface DayIndicatorProps {
+  logData: DayLogData
+  isSelected: boolean
+}
+
+type IndicatorStyle = [ViewStyle, TextStyle]
+
+const DayIndicator: FunctionComponent<DayIndicatorProps> = ({
+  logData,
+  isSelected,
+}: DayIndicatorProps) => {
+  const isToday = DateTimeUtils.isToday(logData.date)
+
+  const applyRiskStyle = ([
```

Comment:
```suggestion
  const applyStyle = ([
```
Similarly, we no longer have a concept of `Risk` on the calendar view for this.